lisa/tools/reboot.py

```diff
@@ -55,9 +54,7 @@ def reboot(self) -> None:
 
         self._log.debug(f"rebooting with boot time: {last_boot_time}")
         try:
-            reboot_result: ExecutableResult = self.node.execute(f"sudo {self.command}")
-            if reboot_result.stderr:
-                self.node.execute_async(f"sudo /usr/sbin/{self.command}")
+            self.node.execute_async(f"sudo /sbin/{self.command}")
```

Comment:
From the PR comments, I cannot understand why remove below logic. The change may have regression.